client-sdk/go/modules/evm/evm.go

```diff
@@ -24,6 +27,12 @@ type V1 interface {
 	// Call generates an EVM CALL transaction.
 	Call(address []byte, value []byte, data []byte, gasPrice []byte, gasLimit uint64) *client.TransactionBuilder
 
+	// Deposit generates a deposit transaction.
+	Deposit(from types.Address, to []byte, amount types.BaseUnits) *client.TransactionBuilder
+
+	// Withdraw generates a withdraw transaction.
+	Withdraw(from []byte, to types.Address, amount types.BaseUnits) *client.TransactionBuilder
```

Comment:
Maybe use some more informative documentation comments (e.g. explain from/to where the deposit/withdraw happens).